modules/SDL.lua

```diff
@@ -153,64 +654,51 @@ end
 -- SDL.RUNNING = 1 Running
 --
 -- SDL.CRASH = -1 Crash
+--
+-- SDL.IDLE = -2 Idle (for remote only)
 function SDL:CheckStatusSDL()
-  local testFile = os.execute ('test -e sdl.pid')
-  if testFile then
-    local testCatFile = os.execute ('test -e /proc/$(cat sdl.pid)')
-    if not testCatFile then
-      return self.CRASH
+  if config.remoteConnection.enabled then
+    local result, data = ATF.remoteUtils.app:CheckAppStatus(config.SDL)
+    if result then
+      if data == remote_constants.APPLICATION_STATUS.IDLE then
+        return self.IDLE
+      elseif data == remote_constants.APPLICATION_STATUS.NOT_RUNNING then
+        return self.STOPPED
+      elseif data == remote_constants.APPLICATION_STATUS.RUNNING then
+        return self.RUNNING
+      end
```

Comment:
@jacobkeeler You are right in general, but Lua developers sometimes omit `return nil` statement because in both cases expression like `local x = some_function()` sets `nil`-value  to variable `x` 
I do not like this too.
Moreover in current case I added error generation because test case is not make sense if unknown SDL status was received. (see fixup commit 3d978ae00fb85325eb6d4dcb580fdf2446931350)